src/marketplaces/in.ts

```diff
@@ -14,6 +17,21 @@ export const IN = new AmazonMarketplace({
   sellerCentralUri: 'https://sellercentral.amazon.in',
   vendorCentralUri: 'https://www.vendorcentral.in',
   webServiceUri: 'https://mws.amazonservices.in',
+  advertising: {
+    countryCode: AmazonMarketplaceAdvertisingCountryCode.IN,
+    region: marketplaceAdvertisingRegions.EU,
+    bids: {
+      sponsoredBrands: {
```

Comment:
How do we know these values are correct?

Reply:
This is the answer I got re bids:

![screenshot-20210907T163618-OXSC0N11](https://user-images.githubusercontent.com/491247/132322268-548a9029-1f64-483f-9bc4-0339faa243d0.png)

Please correct accordingly.

Reply:
Keep in mind that the values are in the smallest possible unit, so need to multiply by 100.

Reply:
@moltar do we have to multiply it by 100? the min bid is 1, if we multiply by 100 it will be 100(1.36 USD) which is a lot in the Indian rupee.

Reply:
Oh wait, maybe the values in the screenshot is already fractional money? Can you confirm that you can bid 1 pice?

Reply:
@moltar I dint find this in docs checked in seller central I can bid 1.

Reply:
Ok, great. So lets update the values then to the ones provided in the screenshot above. Those values were provided by the Amazon team.

Reply:
Done, Thanks!